app/components/Dashboard.jsx

```diff
@@ -45,7 +47,8 @@ class Dashboard extends Component {
       consumerProcessingTimeInMilliseconds,
       consumerPendingDuration,
     } = this.props;
-
+    // Returns ProducerMetrics and ConsumerMetrics with updated state for
```

Comment:
-Same thing as prev comment, the state isn't always updated
-Also add "components" after ConsumerMetrics